### ISSUE TITLE
Missing Thor dependency?

### ISSUE THREAD

**Issue by ftobia** [pact-foundation/pact_broker-client#22]
I think that pact_broker-client-1.13.0 is missing a thor dependency.

Here are my steps to reproduce:
```
90-N6SPFD57-3EV:~ frank.tobia$ docker run --rm -it ruby:2.2 /bin/bash
root@27c3e5c5f857:/# gem install pact_broker-client
Fetching: multi_xml-0.6.0.gem (100%)
Successfully installed multi_xml-0.6.0
Fetching: httparty-0.15.6.gem (100%)
When you HTTParty, you must party hard!
Successfully installed httparty-0.15.6
Fetching: tins-1.15.1.gem (100%)
Successfully installed tins-1.15.1
Fetching: term-ansicolor-1.6.0.gem (100%)
Successfully installed term-ansicolor-1.6.0
Fetching: table_print-1.5.6.gem (100%)
Successfully installed table_print-1.5.6
Fetching: pact_broker-client-1.13.0.gem (100%)
Successfully installed pact_broker-client-1.13.0
6 gems installed
root@27c3e5c5f857:/# pact-broker
/usr/local/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- thor (LoadError)
	from /usr/local/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require'
	from /usr/local/bundle/gems/pact_broker-client-1.13.0/lib/pact_broker/client/cli/custom_thor.rb:1:in `<top (required)>'
	from /usr/local/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require'
	from /usr/local/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require'
	from /usr/local/bundle/gems/pact_broker-client-1.13.0/lib/pact_broker/client/cli/broker.rb:5:in `<top (required)>'
	from /usr/local/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require'
	from /usr/local/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require'
	from /usr/local/bundle/gems/pact_broker-client-1.13.0/bin/pact-broker:2:in `<top (required)>'
	from /usr/local/bundle/bin/pact-broker:23:in `load'
	from /usr/local/bundle/bin/pact-broker:23:in `<main>'
root@27c3e5c5f857:/# gem install thor
Fetching: thor-0.20.0.gem (100%)
Successfully installed thor-0.20.0
1 gem installed
root@27c3e5c5f857:/# pact-broker
Commands:
  pact-broker can-i-deploy -a, --pacticipant=PACTICIPANT -b, --broker-base-url=BROKER_BASE_URL                                      #
  pact-broker create-version-tag -a, --pacticipant=PACTICIPANT -b, --broker-base-url=BROKER_BASE_URL -e, --version=VERSION          # Ad...
  pact-broker help [COMMAND]                                                                                                        # De...
  pact-broker publish PACT_DIRS_OR_FILES ... -a, --consumer-app-version=CONSUMER_APP_VERSION -b, --broker-base-url=BROKER_BASE_URL  # Pu...
  pact-broker version                                                                                                               # Sh...

root@27c3e5c5f857:/#
```

**Comment by bethesque** [comment]
Thanks for the PR. Just to help you level up on your Ruby foo, I'll let you know that I added a '~> 0.20' restrictor for the version, so we don't get hit with any breaking changes from a major release.

**Comment by bethesque** [comment]
I've released version 1.13.1 with the fix.
